src/JoomlaBrowser.php

```diff
@@ -143,10 +143,10 @@ public function installJoomla()
         // Wait while Joomla gets installed
         $this->debug('I wait for Joomla being installed');
         $I->waitForText('Congratulations! Joomla! is now installed.', 60, ['xpath' => '//h3']);
-        $this->debug('Joomla is now installed');
-        $I->see('Congratulations! Joomla! is now installed.',['xpath' => '//h3']);
 		$this->debug('Removing Installation Folder');
 		$I->click(['xpath' => "//input[@value='Remove installation folder']"]);
+		$this->debug('Joomla is now installed');
```

Comment:
This is what we get when we add a waitForElement
![image](https://cloud.githubusercontent.com/assets/3048593/8501798/afe75742-21c6-11e5-8be0-21e57e5484da.png)